app/models/category_list.rb

```diff
@@ -70,7 +70,7 @@ def self.order_categories(categories)
 
   private
 
-  def find_relevant_topics
+  def all_relevant_topics
     @all_topics =
```

Comment:
Is there any reason to set this instance variable anymore since the logic is moved? I'm thinking no, but maybe this is something we want to memoize?

Reply:
It shouldn't be, but also, there was no reason for it to be an instance variable before.

I kept it to not introduce a potentially breaking change, as some plugins might rely on this. 
It is unlikely, but as I did not check, keeping it is safer.